tests/Microsoft.Identity.Test.Unit/ApiConfigTests/AuthorityTests.cs

```diff
@@ -81,7 +75,85 @@ public void WithTenantIdExceptions()
 
             Assert.AreEqual(ex1.ErrorCode, MsalError.TenantOverrideNonAad);
             Assert.AreEqual(ex2.ErrorCode, MsalError.TenantOverrideNonAad);
+        }
+
+        [DataTestMethod]
+        [DynamicData(nameof(TestData.GetAuthorityWithExpectedTenantId), typeof(TestData), DynamicDataSourceType.Method)]
```

Comment:
Cool, this is a first in the repo.

Reply:
It's nice if you need the same Data over and over for Data driven tests